lib/lib.go

```diff
@@ -306,6 +315,16 @@ func NewInstance(ctx context.Context, repoPath string, opts ...Option) (qri *Ins
 		}
 	}
 
+	// if logAll is enabled, turn on logging for all packages. Packages need to be explicitly
+	// enumerated here
+	if o.logAll {
+		allPackages := []string{"qriapi", "qrip2p", "base", "cmd", "config", "dsref", "fsi", "lib", "logbook", "repo"}
```

Comment:
This is going to spit out an _insane_ amount of logs, particularly `qrip2p`, which IIRC spits out a debug message _on every peer connection attempt_. Have you run this & looked at just how verbose the output gets?

If the output is in fact too much, I think the first answer is just to curb the amount of debug messaging in `qrip2p`.

Reply:
It's not ideal, but I don't think it's *too* bad. Right now, `/status` requests are worse than p2p connections, but those will soon get fixed by websockets. I just ran the app for about 10 minutes, and the resulting logfile is 95kb. This means that an hour will get to about 640kb, less than even a Meg. Running for a full day will make a 15mb file. We're going to have to post-filter logs anyway (to get rid of the color bytes), so post-filtering to get rid of p2p connection information won't be too hard to do, and the rest of the log should remain entirely readable. The important thing is that p2p connection attempts only output debug messages every few seconds, so it's not like a sequence of important steps (within the handling of a single api request) will get broken up by a bunch fo p2p messages.